frontend/src/app/libs/auth-pages/sign-in/sign-in.component.ts

```diff
@@ -69,6 +90,20 @@ export class SignInComponent implements OnInit, OnDestroy {
 	}
 
 	public submit(): void {
+		this.recaptchaV3Service.execute('signUpUser')
```

Comment:
until destroy missing?

Reply:
I have added  .pipe(takeUntil(this.destroy$)) here.